server/routes/automation.js

```diff
@@ -17,8 +16,6 @@ const router = Router();
 
 router.get('/:jobID/log', getJobLog, handleError);
 
-router.post('/new', scheduleNewJob, handleError);
```

Comment:
This change and the related ones appear to replace the HTTP API we've designed with an HTTP+GraphQL API. Is that right?

Reply:
Yes. This should've been the approach from the beginning but I read a little too literally into the protocol design document. As far as I can tell, we don't need exposed endpoints for new, cancel, delete, and restart. If we ever want to manage jobs from somewhere other than the ARIA-AT App but still tracked those jobs in the Aria-AT App, then this could be changed.

Reply:
For what it's worth, I don't think you read "too literally;" I think we just spec'd the wrong thing :P I didn't even recognize that until your response just now, so thanks!